cache/cache_test.go

```diff
@@ -87,53 +87,6 @@ func TestFSCache_GetLayer(t *testing.T) {
 				},
 			},
 		},
-		{
```

Comment:
We should add some sad paths here. Especially since something like this https://github.com/aquasecurity/fanal/pull/96/files/26afaa6f737011f7a821e7df4007cda69b5daaa4#r405123771 isn't tested and can break.

Reply:
I added https://github.com/aquasecurity/fanal/pull/96/commits/77f4a81848d5c12820bbd362131df33526962a64